var/spack/repos/builtin/packages/py-xtb/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyXtb(PythonPackage):
+    """Python API for the extended tight binding program package"""
+
+    homepage = "https://xtb-python.readthedocs.org"
+    url = "https://github.com/grimme-lab/xtb-python/releases/download/v22.1/xtb-22.1.tar.gz"
+
+    maintainers = ["awvwgk"]
```

Comment:
FYI. This will overwrite maintainers from the base package.  I'll let Adam comment on whether he likes this idea, but one option is to change this to `maintainers = PythonPackage.maintainers + ["awvwgk"]` so he can also be notified when changes are made.

Reply:
Thanks for the hint, it would be useful to have a function in `spack.package` to deal with this. Maybe something like this:

```diff
-maintainers = [
-    *PythonPackage.maintainers,
-    "user1",
-    "user2",
-]
+maintainer("user1")
+maintainer("user2")
```

Reply:
```suggestion
    maintainers = [
        *PythonPackage.maintainers,
        "awvwgk",
    ]
```

Reply:
This is invalid syntax (at the asterisk).